### PR TITLE
Added more channel flags, added documentation for channel flags.

### PR DESCRIPTION
**Pull request by Red-K0**

### CODE REVIEW

**Comment by Red-K0** [comment]
Ignore both the PR name and commit name, typing anything off remote desktop is not easy 💀

**Comment by Red-K0** [comment]
Added the rest of the channel flags (that I could find) and documented all of them.

**Comment by Red-K0** [comment]
Note: `ApplicationShelfConsent` is empty since I couldn't find anything about what it actually does.

File: NetCord/ChannelFlags.cs [hunk 1]
```diff
@@ -3,6 +3,73 @@
...
+    
+    /// <summary>
+    /// Channel has a global broadcast.
+    /// </summary>
+    Broadcasting = 1 << 14,
+    
+    /// <summary>
+    /// Channel has no download options for media, media channel exclusive.
+    /// </summary>
+    HideMediaDownloadOptions = 1 << 15
```
Comment (KubaZ2) [inline NetCord/ChannelFlags.cs]:
```suggestion
    HideMediaDownloadOptions = 1 << 15,
```

**Review by KubaZ2 — CHANGES_REQUESTED** [review]

**Comment by Red-K0** [comment]
Comma added

**Pull request merged**